<ui version="4.0" >
 <class>upgradeDialog</class>
 <widget class="QDialog" name="upgradeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>System Upgrade</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>170</y>
     <width>389</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QPushButton" name="abortButton" >
      <property name="text" >
       <string>Abort</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="addToQueue" >
      <property name="text" >
       <string>Add to Queue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="goUpgrading" >
      <property name="text" >
       <string>Upgrade Now</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>1</y>
     <width>388</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>A Nice Header goes Here</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="upgradeMessage" >
        <property name="text" >
         <string>Your System can be Upgraded. There are %d
upgradeable packages. You can either upgrade immediately
or add them to the current Queue and process them later.</string>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
        <property name="indent" >
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox" >
      <property name="text" >
       <string>Remember my choice</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
